priv/migrations/add_railway_ipc_consumed_messages.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class AddRailwayIpcConsumedMessages < ActiveRecord::Migration
+class AddRailwayIpcConsumedMessages < ActiveRecord::Migration[4.2]
```

Comment:
Would this cause issues from rails 5 to rails 6?

Reply:
It's mandatory to have a version, any version, in newer Rails.